site/layouts/blog/single.html

```diff
@@ -66,6 +66,12 @@ <h6>Other Posts</h6>
                         {{ end }}
                         {{ end }}
                     </ul>
+                    <h6>Categories</h6>
+                    <ul class="list-unstyled">
+                        {{ range .Params.categories }}
+                    <li class="blog-item__sidebar"><a href="{{ $.Site.BaseURL }}categories/{{ . | urlize }}">{{ . }}</a></li>
```

Comment:
Would you please add a `-` in front of every item similar how it's done for the "Other Posts" lists?

Reply:
There´s some missing pages @ _default and taxonomies is not defined @ the config file. That´s why the pages are empty. 
I´ll fix that as soon as I can.

Reply:
![Captura de tela de 2020-10-21 22-51-56](https://user-images.githubusercontent.com/43054051/96808966-57bcc580-13f0-11eb-855c-6e3fc98ecfd5.png)